apache/superset
# fix(dashboard): Dashboard import commands not correctly replacing charts

### SUMMARY
Similar to https://github.com/apache/superset/pull/22208, the dashboard import commands would not replace charts when the overwrite flag was true, charts would be merged into one dashboard. This change fixes the command in the exact same way as that PR.

Fixes #22127

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
1. Create a dashboard with a chart.
2. Export the dashboard for import later.
3. Modify the dashboard to remove the first chart and add a new one.
4. Import the saved dashboard earlier (via UI OR /api/v1/dashboard/import)
5. Notice both charts are in the dashboard

### ADDITIONAL INFORMATION


- [x] Has associated issue: https://github.com/apache/superset/issues/22127
